modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -891,13 +891,13 @@ class Editor implements EditorObservable {
    * @method resetContent
    * @param {String} initialContent An optional string to use as the initial content of the editor.
    */
-  public resetContent(initialContent?: string) {
+  public resetContent(initialContent?: string, args?: Partial<EditorContent.SetContentArgs>) {
```

Comment:
If you're making this change, you need to update the docs string above and include a change log entry for it. However, this doesn't quite seem right so I'd be interested to know why this was added?

Reply:
Might just have added it because I was adding it everywhere else, to be honest. I'll take it away again.